src/successModal/SuccessModal.story.mdx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { BaseSection } from '../layout/section/baseSection'
+import { SuccessModal } from './index'
+
+<Meta title="Widgets|SuccessModal" />
```

Comment:
```suggestion
<Meta title="Screens|SuccessModal" />
```

I've created a category Screens for all those screens (even though we call them modals)